app/views/contacts/_contact_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class='container'>
+<%= form_for [@company, @contact] do |f| %>
+  <%= f.label :full_name %>
+  <%= f.text_field :full_name %>
+
+  <%= f.label :position %>
+  <%= f.text_field :position %>
+
+  <%= f.label :email %>
+  <%= f.text_field :email %>
+
+  <%= hidden_field_tag :company_id, @company.id %><br>
```

Comment:
i suggest nesting your routes rather than passing hidden data here. users can manipulate this data